src/plugins/loan-channel-query-map.js

```diff
@@ -56,17 +56,17 @@ export default {
 				{
 					id: 28,
 					url: 'kiva-u-s',
-					queryParams: 'status=fundRaising&country=US&distributionModel=direct',
+					queryParams: 'status=fundRaising&country=US,GU,VI,PR&distributionModel=direct',
 					algoliaParams: 'countries=North%20America%20%3E%20United%20States',
 					flssLoanSearch: {
-						countryIsoCode: ['US'],
+						countryIsoCode: ['US,GU,VI,PR'],
```

Comment:
I think this may fix an outstanding bug with what loans are shown for the Kiva US category...